packages/discord.js/src/managers/GuildMemberRoleManager.js

```diff
@@ -174,6 +160,45 @@ class GuildMemberRoleManager extends DataManager {
     }
   }
 
+  /**
+   * Modifies the roles of the member.
+   * @param {RoleResolvable|RoleResolvable[]|Collection<Snowflake, Role>} rolesToAdd The roles to add
+   * @param {RoleResolvable|RoleResolvable[]|Collection<Snowflake, Role>} rolesToRemove The roles to remove
+   * @param {string} [reason] Reason for modifying the roles
+   * @returns {Promise<GuildMember>}
+   */
+  async modify(rolesToAdd, rolesToRemove, reason) {
```

Comment:
The JSDocs parameters are inconsistent with the typings, there is no `RoleResolvable` in typings but it is marked as a supported type in JSDocs.